src/clear-all-refinements/index.ts

```diff
@@ -1,8 +1,15 @@
-import { tag, Tag } from '@storefront/core';
+import { tag, StoreSections, Tag } from '@storefront/core';
 
 @tag('gb-clear-all-refinements', require('./index.html'))
 class ClearAllRefinements {
-  onClick = () => this.actions.resetRefinements(true);
+  onClick = () => {
+    switch (this.props.storeSection) {
```

Comment:
Instead of doing this check every time a click happens, set this up in `init` using a state property

Reply:
For this one and the `clear-refinement` one, if I were to have *action* as a property in *state*, would you prefer it be a string with the action key or the entire action?

Reply:
I think the action itself makes sense